CHANGES.rst

```diff
@@ -479,6 +479,12 @@ Bug fixes
 
 - ``astropy.nddata``
 
+  - Initializing ``NDDataArray`` from another instance now sets ``flags`` as
+    expected and no longer fails when ``uncertainty`` is set [#4129].
+    Initializing an ``NDData`` subclass from a parent instance
+    (eg. ``NDDataArray`` from ``NDData``) now sets the attributes other than
+    ``data`` as it should [#4137].
+
```

Comment:
Please also include the string `[skip ci]` in the commit message for changes that only make minor doc formatting changes.